### DISCUSSION TITLE
Update existing .json instead of completely overriding it

### DISCUSSION THREAD

**Discussion by Xosrov** [structurizr/cli#131] [Q&A] [answered]
### Question

I've created some diagrams that are a little complicated, so I've had to disable autolayout and move them around manually using `structurizr/lite` for better readability. 

When I add a new element to the view and refresh the lite webpage, all other elements still have their original placements. I was wondering if something similar would be possible in the cli version; since right now, using `export -format json` on a .dsl file, completely overrides the previous .json and all its manual placements.  

I couldn't find anything on my own. Is this functionality possible, or is it not implemented here yet?

Thanks.

**Comment by simonbrowndotje** [comment, marked as answer]
The CLI does this automatically when you `push` a workspace to a cloud/on-premises workspace, but it's not available as a standalone feature.

**Reply by Mitriay** [reply]
Hi, i am facing the same problem using cli push command. My structurizr-lite diagrams previously manually formatted have different look in structurizr on-premise after "push" command.

**Reply by simonbrowndotje** [reply]
Please see https://github.com/structurizr/cli/discussions/119#discussioncomment-5968439 for an existing discussion on this topic.
